app/views/display_tasks/_form.html.erb

```diff
@@ -0,0 +1,14 @@
+<fieldset>
```

Comment:
Nice use of the partial! If you pull out the method and url into variables, you could put the whole `form_for` inside of this partial as well